website/docs/reference/administrative-commands/createIndexes.mdx

```diff
@@ -0,0 +1,123 @@
+---
+sidebar_position: 1
+description: Create one or more indexes on a collection to improve query performance and enable certain query features.
+---
+
+# `createIndexes`
+
+import CodeBlock from '@theme/CodeBlock';
+import CreateIndex from '!!raw-loader!./create-indexes/1-create-simple-index.request.js';
+import CreateUniqueIndex from '!!raw-loader!./create-indexes/2-create-unique-index.request.js';
+import CreateCompoundIndex from '!!raw-loader!./create-indexes/3-create-compound-index.request.js';
+import CreatePartialIndex from '!!raw-loader!./create-indexes/4-create-partial-index.request.js';
+import CreateSparseIndex from '!!raw-loader!./create-indexes/5-create-sparse-index.request.js';
+
+<!-- Category for Command "Administrative, Diagnostic, and Utility Commands" should be organized per folder as present in the compatibility document -->
+
+<!-- Short description of the command and its usage. -->
+
+Create one or more indexes on a given collection to improve query performance or enable certain query features (e.g., full-text, vector, and TTL indexes).
```

Comment:
create**s**, enable**s**. There is no actor

Reply:
This comment is forgotten. Missing enable**s**